src/util-file.c

```diff
@@ -863,14 +883,16 @@ static File *FileOpenFile(FileContainer *ffc, const StreamingBufferConfig *sbcfg
         ff->flags |= FILE_USE_DETECT;
     }
 
-    if (!(ff->flags & FILE_NOMD5) || g_file_force_md5) {
-        ff->md5_ctx = SCMd5New();
-    }
-    if (!(ff->flags & FILE_NOSHA1) || g_file_force_sha1) {
-        ff->sha1_ctx = SCSha1New();
-    }
-    if (!(ff->flags & FILE_NOSHA256) || g_file_force_sha256) {
-        ff->sha256_ctx = SCSha256New();
+    if (!g_disable_hashing) {
```

Comment:
Did you see this be necessary or is this more to be on the safe side?

Reply:
In master this section is wrapped in an `ifdef HAVE_NSS` guard, so it makes sense to wrap this in a `g_disable_hashing` guard.

Reply:
My expectation would that this is redundant. I would exist the `FILE_NO<hash>` flags to be set and the `g_file_force_<hash>` to be unset. I think I've confirmed the latter, but not yet sure about the former.

Reply:
Yes, it appears to be redundant in my testing.